monitoring/monitorlib/kml.py

```diff
@@ -0,0 +1,101 @@
+# Module to parse KML file.
+
+from pykml import parser
+import re
+
+
+KML_NAMESPACE = {"kml":"http://www.opengis.net/kml/2.2"}
+
+def get_kml_root(kml_path):
+    kml = parser.parse(kml_path)
+    return kml.getroot()
+
+
+def get_folders(root):
+    return root.Document.Folder.Folder
+
+
+def get_polygon_speed(polygon_name):
+    """Returns speed unit within a polygon."""
+    result = re.search(r"\(([0-9.]+)\)", polygon_name)
+    return float(result.group(1)) if result else None
+
+def get_folder_details(folder_elem):
+    speed_polygons = {}
+    alt_polygons = {}
+    operator_location = {}
+    coordinates = ''
+    for placemark in folder_elem.xpath('.//kml:Placemark', namespaces=KML_NAMESPACE):
+        placemark_name = str(placemark.name)
+        polygons = placemark.xpath('.//kml:Polygon', namespaces=KML_NAMESPACE)
+
+        if placemark_name == 'operator_location':
+            operator_location = {
+                'lng': folder_elem.xpath(
+                    './/kml:Placemark/kml:LookAt/kml:longitude', namespaces=KML_NAMESPACE)[0],
```

Comment:
[The LookAt tag](https://developers.google.com/kml/documentation/kmlreference#lookat) defines where the camera should be looking when the user double-clicks this Placemark in Google Earth, but I think we want to extract the actual location of the Placemark itself instead (e.g., 39.078915 latitude instead of 39.077887 latitude for test_flight_north)